pkg/utils/kubernetes/service.go

```diff
@@ -20,12 +20,17 @@ import (
 	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
 )
 
+// FQDNForService returns the fully qualified domain name of a service with the given name and namespace
```

Comment:
```suggestion
// FQDNForService returns the fully qualified domain name of a service with the given name and namespace.
```

Can you add a unit test for this function?